skimage/segmentation/tests/test_chanvese.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from numpy.testing import assert_array_equal, assert_raises
+from skimage.segmentation import chan_vese
+
+
+def test_chan_vese_simple_shape():
+    img = np.zeros((10, 10))
+    img[3:6, 3:6] = np.ones((3, 3))
+    result = chan_vese(img, mu=0.0, tol=1e-8)[1]*1.0
+    assert_array_equal(result, img)
+
+
+def test_chan_vese_remove_noise():
+    img = np.zeros((10, 10))
+    ref = np.zeros((10, 10))
+    img[1:6, 1:6] = np.array([[0, 1, 1, 1, 0],
+                              [1, 1, 1, 1, 1],
+                              [1, 1, 1, 1, 1],
+                              [1, 1, 1, 1, 1],
+                              [0, 1, 1, 1, 0]])
+    ref[:, :] = img[:, :]
+    img[8, 3] = 1
+    result = chan_vese(img, mu=1.2, tol=1e-3, maxiter=100, dt=10,
+                       starting_level_set="disk")
+    assert_array_equal(result[1]*1.0, ref)
+
+
+def test_chan_vese_gap_closing():
+    img = np.zeros((10, 10))
+    ref = np.zeros((10, 10))
+    img[4:7, :] = np.ones((3, 10))
+    ref[:, :] = img[:, :]
+    img[:, 6] = np.zeros((10))
+    result = chan_vese(img, mu=1.2, tol=1e-3, maxiter=100, dt=1,
+                       starting_level_set="disk")
+    assert_array_equal(result[1]*1.0, ref)
+
+
+def test_chan_vese_incorrect_level_set_size():
```

Comment:
As far as you check also if string is valid, should be named as, for example, `test_chan_vese_incorrect_level_set_type`